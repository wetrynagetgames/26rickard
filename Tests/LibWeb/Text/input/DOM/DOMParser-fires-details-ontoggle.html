<!DOCTYPE html>
<script src="../include.js"></script>
<script>
    asyncTest(done => {
        new DOMParser().parseFromString("<details open>", "text/html").querySelector("details").ontoggle = function(e) {
            println(`DOMParser toggle event is ${e.target.open}`);
            done();
        };
    });
</script>
