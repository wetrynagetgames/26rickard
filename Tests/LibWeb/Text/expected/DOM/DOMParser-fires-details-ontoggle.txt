DOMParser toggle event is true
